Core/scripts/dirac-install.py

```diff
@@ -741,17 +741,20 @@ def getTarsLocation(self, project, module=None):
       return S_OK(sourceUrl)
     return S_ERROR("Don't know how to find the installation tarballs for project %s" % project)
 
-  def getDiracOsLocation(self, project=None):
+  def getDiracOsLocation(self, project=None, diracosFromDIRAC=False):
     """
     Returns the location of the DIRAC os binary for a given project for example: LHCb or DIRAC, etc...
 
     :param str project: the name of the project
+    :param bool diracosFromDIRAC: where to take diracos
```

Comment:
:param bool diracosDefault: flag to take diracos distribution from the default location